From fbf6c484c7aeafa012a35f1b9b17a12c751909cc Mon Sep 17 00:00:00 2001
From: Arian <arian.kulmer@web.de>
Date: Thu, 27 Mar 2025 05:20:20 +0000
Subject: [PATCH 4/4] Change com.android.camera.action.REVIEW to
 android.provider.action.REVIEW This is the standard intent for review actions

Signed-off-by: zenin1504 <kayozzprojects@yandex.com>
---
 smali/com/android/camera/w3.smali | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/smali/com/android/camera/w3.smali b/smali/com/android/camera/w3.smali
index 1cfa2eb69..c3dcd8016 100644
--- a/smali/com/android/camera/w3.smali
+++ b/smali/com/android/camera/w3.smali
@@ -1136,7 +1136,7 @@
 
     move-result p0
 
-    const-string v0, "com.android.camera.action.REVIEW"
+    const-string v0, "android.provider.action.REVIEW"
 
     if-eqz p0, :cond_0
 
@@ -1197,7 +1197,7 @@
 
     const-string p1, "GalleryHelper"
 
-    const-string v1, "gotoGallery: com.android.camera.action.REVIEW"
+    const-string v1, "gotoGallery: android.provider.action.REVIEW"
 
     invoke-static {p1, v1, p0}, Lcom/android/camera/log/Log;->d(Ljava/lang/String;Ljava/lang/String;[Ljava/lang/Object;)V
 
-- 
2.39.5
